test/python/pulse/test_calibrationbuilder.py

```diff
@@ -0,0 +1,110 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Test the RZXCalibrationBuilderNoEcho."""
+
+from math import pi, erf, ceil
+
+import numpy as np
+
+from qiskit import circuit, schedule
+from qiskit.transpiler import PassManager
+from qiskit.test import QiskitTestCase
+from qiskit.pulse import (
+    Play,
+    Delay,
+    ShiftPhase,
+    ControlChannel,
+    DriveChannel,
+    GaussianSquare,
+)
+from qiskit.transpiler.passes.scheduling.calibration_creators import (
+    RZXCalibrationBuilderNoEcho,
+)
+from qiskit.test.mock import FakeAthens
+
+
+class TestCalibrationBuilder(QiskitTestCase):
+    """Test the Calibration Builder."""
+
+    def setUp(self):
+        super().setUp()
+        self.backend = FakeAthens()
+        self.configuration = self.backend.configuration()
+        self.defaults = self.backend.defaults()
+        self.inst_map = self.defaults.instruction_schedule_map
```

Comment:
```suggestion
        self.inst_map = self. backend.defaults().instruction_schedule_map
```
Looks like you do not need the configuration and the defaults.